packages/api-client/src/views/Request/Request.vue

```diff
@@ -106,6 +107,8 @@ onMounted(() => {
   events.cancelRequest.on(cancelRequest)
 })
 
+useOpenApiWatcher()
```

Comment:
This feels like a strange place to enable the watcher. Should be done at the App level as its a background state watcher and not specific to a single page.

Reply:
Its only used on the request page though